Mayhemfile

```diff
@@ -0,0 +1,15 @@
+# Namespaced project name that the target belongs to
+project: jbgreenh/sndfilter
+
+# Target name (should be unique within the project)
+target: sndfilter
+
+# List of commands used to test the target
+cmds:
+
+    # Command used to start the target, "@@" is the input file
+    # (when "@@" is omitted Mayhem defaults to stdin inputs)
+    - cmd: /sndfilter @@ <reverb> <5> <longreverb1>
```

Comment:
We do not currently support multiple commands in the Mayhemfile at the moment. Please either omit the extra commands, or split this Mayhemfile into multiple Mayhemfiles (see here: https://community.forallsecure.com/t/error-the-mayhem-for-code-scan-detected-the-mayhem-run-for-your-target-was-unsuccessful-and-multiple-docker-files-in-the-repository/350/5).